cmd/minikube/cmd/start.go

```diff
@@ -464,7 +464,7 @@ func showKubectlInfo(kcs *kubeconfig.Settings, k8sVersion string, machineName st
 		out.Ln("")
 		out.WarningT("{{.path}} is version {{.client_version}}, which may have incompatibilites with Kubernetes {{.cluster_version}}.",
 			out.V{"path": path, "client_version": client, "cluster_version": cluster})
-		out.T(style.Tip, "Want kubectl {{.version}}? Try 'minikube kubectl -- get pods -A'", out.V{"version": k8sVersion})
```

Comment:
This is intentional -- it's something users should absolutely see.

Reply:
I think `out.Infof` is always shown?

Reply:
Sorry, I missed that. So what's the difference between out.T and out.Infof then? Based on the naming of the functions, it isn't obvious why one would break this test.

Reply:
so `out.T` could probably be better named to `out.Step` -- when I implemented JSON output I basically distinguished anything sent to `out.T` as a distinct progress step for `minikube start`, and anything sent to `out.Info` as an informational message that wouldn't be a distinct step in the process.

The test basically ensures that all things sent to `out.T` are distinctly numbered steps, so if someone adds a new step via `out.T` it ensures they will add that step to the JSON output registry & JSON output will still be correct.

Reply:
I could keep this as `out.T` and add a `CheckKubectlVersion` step to the registry as an alternate fix, but this struck me as more of an informational message.

Reply:
I think your idea of renaming `out.T` to `out.Step` would go a long way to making this less confusing for current and future readers. If we don't address this confusion, I'm sure this kind of issue will pop up again.

Reply:
Cool, opened an issue to track that: https://github.com/kubernetes/minikube/issues/9664